I/ITKIOWrapper/build_tarballs.jl

```diff
@@ -1,59 +1,75 @@
 # Note: this script will require BinaryBuilder.jl v0.3.0 or greater
 using BinaryBuilder, Pkg
 name = "ITKIOWrapper"
-version = v"1.0.0"  # Update this to your package version
-
-# The following lines should be updated to reflect the actual URL and revision of your ITKWrapper package
-
+version = v"2.0.0"
 # Collection of sources required to build ITKWrapper
 sources = [
     DirectorySource("./src"),
 ]
-uuid = Base.UUID("a83860b7-747b-57cf-bf1f-3e79990d037f")
-delete!(Pkg.Types.get_last_stdlibs(v"1.6.3"), uuid)
 
+uuidopenssl = Base.UUID("458c3c95-2e84-50aa-8efc-19380b2a3a95")
+delete!(Pkg.Types.get_last_stdlibs(v"1.12.0"), uuidopenssl)
+delete!(Pkg.Types.get_last_stdlibs(v"1.13.0"), uuidopenssl)
 #needed for libjulia_platforms and julia_versions
 include("../../L/libjulia/common.jl")
 
+#filter julia versions to include only Julia >= 1.10 for LTS
+julia_versions = filter(v-> v >= v"1.10", julia_versions)
+
 # Bash recipe for building across all platforms
 script = raw"""
 export CXXFLAGS="-I${includedir}/julia $CXXFLAGS"
 export CFLAGS="-I${includedir}/julia $CFLAGS"
 mkdir -p build/
-cmake -B build -S . \
-    -DCMAKE_INSTALL_PREFIX=${prefix} \
-    -DCMAKE_BUILD_TYPE=Release \
-    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
-    -DBUILD_SHARED_LIBS:BOOL=ON
+
+# Different CMake configuration for macOS
+if [[ "${target}" == *-apple-* ]]; then
+    cmake -B build -S . \
+        -DCMAKE_INSTALL_PREFIX=${prefix} \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+        -DBUILD_SHARED_LIBS:BOOL=ON \
+        -DCMAKE_POSITION_INDEPENDENT_CODE=ON \
+        -DCMAKE_OSX_DEPLOYMENT_TARGET=10.10
+else
+    cmake -B build -S . \
+        -DCMAKE_INSTALL_PREFIX=${prefix} \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+        -DBUILD_SHARED_LIBS:BOOL=ON \
+        -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+fi
```

Comment:
```suggestion
if [[ "${target}" == x86_64-apple-* ]]; then
    MACOSX_DEPLOYMENT_TARGET=10.10
fi
cmake -B build -S . \
    -DCMAKE_INSTALL_PREFIX=${prefix} \
    -DCMAKE_BUILD_TYPE=Release \
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
    -DBUILD_SHARED_LIBS:BOOL=ON \
    -DCMAKE_POSITION_INDEPENDENT_CODE=ON
```
but I'm missing why you need to set the SDK to 10.10 at all, since we're using 10.12, an explanation would be most most welcome.

Reply:
Setting it to 10.10 ensures broader compatibility with older macOS  and a lower deployment target allows the binary to run on older versions

Reply:
[Julia itself requires macOS 10.14](https://julialang.org/downloads/#supported_platforms), so I'm not sure what you gain by doing that.

Reply:
interesting, so removing the condition at all will certainly set the default to 10.12 I believe, I will try that locally, and commit changes if i succeed.

Reply:
https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/bc1c293e367714537136d22ece9d81a82731b009/src/Runner.jl#L116-L145 Actually it may be 10.10 _already_.  Either way, I don't think you need to set anything older than whatever is already the (old) default.